packages/libclang/libclang.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata minClientVersion="2.12">
+    <id>libclang</id>
+    <version>8.0.0</version>
+    <authors>Microsoft and Contributors</authors>
+    <owners>Microsoft and Contributors</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <projectUrl>https://github.com/llvm-mirror/clang</projectUrl>
```

Comment:
The license should definitely match the original binaries, but it might make sense to use the clangsharp project url, instead of theirs. Thoughts?

Reply:
There is also the repository url...

Reply:
Actually, looks like NuGet lists `projectUrl` as
> A URL for the package's home page, often shown in UI displays as well as nuget.org.

`repository` is listed as:
> Repository metadata, consisting of four optional attributes: type and url (4.0+), and branch and commit (4.6+). These attributes allow you to map the .nupkg to the repository that built it, with the potential to get as detailed as the individual branch or commit that built the package. This should be a publicly available url that can be invoked directly by a version control software. It should not be an html page as this is meant for the computer. For linking to project page, use the projectUrl field, instead.

@mjsabby, what do you think of making `projectUrl` be `https://github.com/microsoft/ClangSharp` and `repository` be: `<repository type="git" url="https://github.com/llvm-mirror/clang" branch="release_80" />`?

I think that would satisfy both who created the package and where the sources were built from.

Reply:
Sounds good.

Reply:
Fixed that up, and also updated the nuspec to:
1. Explicitly list the content files (so a failure occurs if you forget to copy over the native library)
2. Reuse the same LICENSE.TXT file from the `packages/libclang` directory, so we don't need 12 copies of it.